project/home/urls.py

```diff
@@ -14,7 +14,8 @@
     path('accounts/logout/', CustomLogoutView, name='logout'),  # Use your custom logout view
     path('accounts/register/', register, name = "register"),
     path('accounts/profile/', views.userPage, name = "user_page"),
-
+    path('accounts/update_password/', views.update_password, name = "update_password"),
+    path('accounts/update_account/', views.update_password, name = "update_account"),
```

Comment:
Should probably be views.update_account for the update_account path.

Reply:
Thanks David for catching that. I fixed it!